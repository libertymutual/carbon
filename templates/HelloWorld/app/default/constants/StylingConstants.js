// Styling Constants
export const COLOR_LIGHTGRAY = 'lightgray';
