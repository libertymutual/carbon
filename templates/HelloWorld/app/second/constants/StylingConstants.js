// Styling Constants
export const COLOR_LIGHTBLUE = 'lightblue';
