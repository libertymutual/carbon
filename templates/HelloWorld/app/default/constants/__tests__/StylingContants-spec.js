import 'react-native';
import React from 'react';
import * as StylingConstants from '../StylingConstants';

describe('StylingConstants tests', () => {

  test('COLOR_LIGHTGRAY returns correct value', () => {
    expect(StylingConstants.COLOR_LIGHTGRAY).toMatchSnapshot();
  });
});
