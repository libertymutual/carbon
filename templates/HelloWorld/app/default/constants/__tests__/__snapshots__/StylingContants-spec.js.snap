// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StylingConstants tests COLOR_LIGHTGRAY returns correct value 1`] = `"lightgray"`;
